### PR TITLE
[Whisper] Elixir implementation PoC from file

### PR DESCRIPTION
**Pull request by rafalskorupa**

### CODE REVIEW

File: lib/chirp_cockroach/audio/whisper.ex [hunk 1]
```diff
@@ -0,0 +1,43 @@
+defmodule ChirpCockroach.Audio.Whisper do
```
Comment (rafalskorupa) [inline lib/chirp_cockroach/audio/whisper.ex:1]: Only used for caching models/serving - Nx provides a way to define it directly from supervision tree, but it didn't work for me out of the box

**Comment by erszcz** [comment]
@rafalskorupa This PR is marked as a draft. Do I get it right that it's only a technology preview not intended for merging? Is this (directly or parts of) incorporated into the subsequent PRs sent after this one?

**Comment by rafalskorupa** [comment]
@erszcz The whole idea and implementation were incorporated in IRC PR that is merged, so I'm closing that PR.

**Comment by erszcz** [comment]
Sounds good, thanks 👍

**Pull request closed without merging**